server/cmd/server/.env

```diff
@@ -12,6 +12,8 @@ MULTIVERSX_PROXY="https://testnet-gateway.multiversx.com"
 HEADER_VERIFIER_SC_ADDRESS="erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"
 # ESDT Safe address on MultiversX to execute the transactions
 ESDT_SAFE_SC_ADDRESS="erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"
+# Change epoch validators set address on MultiversX to execute the transactions
+CHANGE_VALIDATORS_SC_ADDRESS="erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"
```

Comment:
I don't see this really necessary but it's fine..

Reply:
Why not necessary? We need those

Reply:
It will be the same address as `HEADER_VERIFIER_SC_ADDRESS`. But it's fine, maybe in the future will be a different contract